### PR TITLE
Jormungandr: migrate from nose to pytest

### PR DESCRIPTION
**Pull request by antoine-de**
pytest is way nicer than nose and is already used in the jormungandr test and tyr integration tests

for the moment it's a raw migration without using all pytest capabilities (like the nice test fixtures)

we'll be able to do this afterward.

There is the same number of tests run (we won 1 test because it has the same name as another test :wink: )

### CODE REVIEW

File: source/jormungandr/tests/departure_board_tests.py [hunk 1]
```diff
@@ -271,7 +273,8 @@ def test_partial_terminus(self):
         assert response["stop_schedules"][0]["route"]["id"] == "A:1"
         assert len(response["stop_schedules"][0]["date_times"]) == 2
         assert response["stop_schedules"][0]["date_times"][0]["links"][0]["type"] == "notes"
-        assert response["stop_schedules"][0]["date_times"][0]["links"][0]["id"] == "destination:16710925402715739122"
+        assert response["stop_schedules"][0]["date_times"][0]["links"][0][
```
Comment (pbougue) [inline source/jormungandr/tests/departure_board_tests.py]: weird line return

File: source/jormungandr/tests/departure_board_tests.py [hunk 2]
```diff
@@ -453,11 +454,13 @@ def test_departures(self):
         is_valid_departures(response["departures"])
 
         assert len(response["departures"][0]["stop_date_time"]["additional_informations"]) == 1
-        assert response["departures"][0]["stop_date_time"]["additional_informations"][0] == "date_time_estimated"
+        assert response["departures"][0]["stop_date_time"]["additional_informations"][
```
Comment (pbougue) [inline source/jormungandr/tests/departure_board_tests.py]: line return

File: source/jormungandr/tests/departure_board_tests.py [hunk 3]
```diff
@@ -453,11 +454,13 @@ def test_departures(self):
...
+                   0] == "date_time_estimated"
         assert response["departures"][0]["stop_date_time"]["data_freshness"] == "base_schedule"
 
         assert len(response["departures"][1]["stop_date_time"]["additional_informations"]) == 1
-        assert response["departures"][1]["stop_date_time"]["additional_informations"][0] == "on_demand_transport"
+        assert response["departures"][1]["stop_date_time"]["additional_informations"][
```
Comment (pbougue) [inline source/jormungandr/tests/departure_board_tests.py]: line return

File: source/jormungandr/tests/pytest.ini [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = --doctest-modules
+python_files=*tests*.py
+python_classes=Test
+python_functions=*test*
```
Comment (pbougue) [inline source/jormungandr/tests/pytest.ini]: new line

File: source/jormungandr/tests/journey_common_tests.py [hunk 5]
```diff
@@ -368,9 +368,13 @@ def test_journeys_wheelchair_profile(self):
         self.is_valid_journey_response(response, query)
 
         assert(len(response['journeys']) == 2)
+
+        def compare(l1, l2):
```
Comment (pbougue) [inline source/jormungandr/tests/journey_common_tests.py:372]: might deserve to be in test utils

**Review by pbougue — APPROVED** [review]
minor comments

File: source/jormungandr/tests/journey_common_tests.py (hunk 5 above)
Comment (antoine-de) [reply source/jormungandr/tests/journey_common_tests.py:372]: since it's used only there I more for letting it there :stuck_out_tongue_winking_eye:

**Comment by antoine-de** [comment]
the test seems to be correcly run

https://ci.navitia.io/job/navitia_pull_request_build/4047/testReport/ >> 2414 tests
https://ci.navitia.io/job/navitia_pull_request_build/4049/testReport/ >> 2416 tests

the difference is the integration test wrongly named (run twice, once for marshall, one for serpy)

File: source/jormungandr/tests/pytest.ini (hunk 4 above)
Comment (antoine-de) [reply source/jormungandr/tests/pytest.ini]: done

**Comment by pbougue** [comment]
Jenkins, test this please

**Pull request merged**